fractals.json

```diff
@@ -0,0 +1,202 @@
+{
+    "scales": [
+        { "scale": 1, "type": "volcanic", "ar": 0 },
```

Comment:
I could also add the lobby (Mistlock Observatory) in this list, that would also align the index with the scale 🤔